test/beacon/Voucher.test.ts

```diff
@@ -275,6 +283,29 @@ describe('Voucher', function () {
             iexecPocoInstance.balanceOf(requester.getAddress());
 
         it('Should match orders with full sponsored amount', async () => {
+            const mockOrder = createMockOrder();
+            const appOrder = { ...mockOrder, app: app, appprice: appPrice };
+            const datasetOrder = {
+                ...mockOrder,
+                dataset: dataset,
+                datasetprice: datasetPrice,
+            };
+            const workerpoolOrder = {
+                ...mockOrder,
+                workerpool: workerpool,
+                workerpoolprice: workerpoolPrice,
+            };
+            const requestOrder = { ...mockOrder, requester: requester.address };
+            // Should match orders with low app order volume
+            // Set volumes
+            appOrder.volume = 2; // smallest unconsumed volume among all orders
+            datasetOrder.volume = 3;
+            workerpoolOrder.volume = 4;
+            requestOrder.volume = 5;
+            const expectedVolume = 2;
```

Comment:
No need to do this, just use the same volume everywhere. Volume compute is out of the Voucher's scope.
The global `volume = 3` would be enough.

Reply:
Done here : [set volume as global in test and compute dealPrice without dataset](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/19/commits/34d403704fa265e44daa5a64128b1a4ac4a69fe5)